lib/one_time_password/base.rb

```diff
@@ -12,6 +12,11 @@ def rotp
 
     private
 
-    attr_reader :issuer
+    attr_reader :issuer, :secret
+
+    def encode_secret(secret)
+      return ROTP::Base32.encode(secret) if secret
+      SECRET
+    end
```

Comment:
not blocking and probably a stylistic thing. i think i would of made this return nil if `secret` is nil. then the caller can do the following
```ruby
@secret = encode_secret(secret) || SECRET
```

Reply:
👍🏽 done